routes/file.js

```diff
@@ -133,25 +133,26 @@ module.exports = ( request, reply ) => {
 			// the original version back.
 			do_compress = false;
 		}
+		do_compress = true;
```

Comment:
Did you mean to leave this here? It looks like a stray debugging/testing statement.